.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+
+on:
+  release:
```

Comment:
We don't do releases, this should be on push and only for the master branch. (I don't think we will need pushing for feature branches so we can leave that for now)

Reply:
Oops I didn't see that there are no releases. So it's gonna be pretty easy to just build and publish with the `latest` tag.

Also without tags and proper releases we cannot be sure that the app is working on a `docker pull` if master is broken for a given time.

Reply:
excalidraw.com is on continuous deployment from master, so hopefully the docker image won't be broken :)

Reply:
🤞